AssetInformationListener/Factories/EntityFactory.cs

```diff
@@ -0,0 +1,31 @@
+using AssetInformationListener.Domain;
+using AssetInformationListener.Infrastructure;
+
+namespace AssetInformationListener.Factories
+{
+    public static class EntityFactory
+    {
+        public static DomainEntity ToDomain(this DbEntity databaseEntity)
+        {
+            return new DomainEntity
+            {
+                // TODO - Implement factory method fully
```

Comment:
Remove comment as complete

Reply:
It will be when the next ticket is done. This is just setting up the application shell.